conditional/blueprints/intro_evals.py

```diff
@@ -31,11 +31,24 @@ def display_intro_evals(internal=False, user_dict=None):
     log = logger.new(request=request, auth_dict=user_dict)
     log.info('Display Intro Evals Listing')
 
-    # get user data
-    def get_fid_cm_count(member_id):
-        return len([a for a in FreshmanCommitteeAttendance.query.filter(
-            FreshmanCommitteeAttendance.fid == member_id)
-            if CommitteeMeeting.query.filter(CommitteeMeeting.id == a.meeting_id).first().approved])
+    def get_fid_cm(member):
+        c_meetings = [{
+            "uid": cm.uid,
+            "timestamp": cm.timestamp,
+            "committee": cm.committee
+        } for cm in CommitteeMeeting.query.join(
+            FreshmanCommitteeAttendance,
+            FreshmanCommitteeAttendance.meeting_id == CommitteeMeeting.id
+            ).with_entities(
+                FreshmanCommitteeAttendance.uid,
+                CommitteeMeeting.timestamp,
+                CommitteeMeeting.committee
+                ).filter(
+                    CommitteeMeeting.timestamp > start_of_year(),
+                    FreshmanCommitteeAttendance.uid == member.uid,
```

Comment:
The function is `get_fid_cm` and it doesn't look like you're referring to an `fid` at all. What is the type of `member`?

Reply:
Looking below, it looks like you're passing an `fid` as member (which is an int), so it wouldn't have a `uid` field